apps/internal/oauth/ops/authority/authority.go

```diff
@@ -307,8 +272,6 @@ func (c Client) UserRealm(ctx context.Context, authParams AuthParams) (UserRealm
 	err := c.Comm.JSONCall(
 		ctx,
 		endpoint,
-		// TODO(jdoak): not thrilled about this, because all calls should have this but
```

Comment:
I think we add Correlation Id to all the calls that are part of a user flow for telemetry. I see thats what we do in MSAL Python. It was added here : https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/103 
Could not find a spec though. @henrik-me thoughts ?